arlas-tests/src/test/java/io/arlas/server/rest/tag/TagIT.java

```diff
@@ -249,4 +249,6 @@ public void testUntag() throws InterruptedException {
         LOGGER.error("Assertion failed " + tries + " times, we give up.");
         return false;
     }
+
+    //TODO test with columns filter optionnaly
```

Comment:
This can be skipped as the tagger is removed from future version and not used in this version for the targeted deployment.

Reply:
With Matthieu we thought we could possibly add some tests if we had time. But if this won't be used, no reason to loose time on it.